dbusmock/templates/networkmanager.py

```diff
@@ -26,8 +26,9 @@
 
 
 BUS_NAME = 'org.freedesktop.NetworkManager'
-MAIN_OBJ = '/org/freedesktop/NetworkManager'
-MAIN_IFACE = 'org.freedesktop.NetworkManager'
+MAIN_OBJ = '/org/freedesktop'
```

Comment:
That looks wrong, this is almost certainly not a valid object path. It would also break backwards compatibility?